tests/test_netstats.py

```diff
@@ -23,10 +23,10 @@ def test_get_prop_type(value):
     """
```

Comment:
I seem to have gotten my branches mixed up and added netstats timer additions in this pull request as well. Did you want me to remove it from this pull request or close PR #484 which has identical changes?

Reply:
Removing it from this PR would be ideal I think